app/Http/routes.php

```diff
@@ -79,6 +79,9 @@
 Route::get('/community/slack', ['as' => 'slack', 'uses' => 'CommunityController@getSlack']);
 Route::post('/community/slack/agree', ['as' => 'slack.agree', 'uses' => 'CommunityController@postSlackAgree']);
 
+Route::get('/mp/{match_id}', ['as' => 'multiplayer.match', 'uses' => 'MultiplayerController@match']);
+Route::get('/mp/{match_id}/history', ['as' => 'multiplayer.match.history', 'uses' => 'MultiplayerController@matchHistory']);
```

Comment:
Ideally we want the full URLs to be canonical and the shorter ones redirecting to them, for example:
`/b/{id}` redirecting to `/beatmaps/{id}`
`/s/{id}` redirecting to `/beatmapsets/{id}`
`/mp/{id}` redirecting to `/matches/{id}`

(I confirmed this with peppy just to be sure)